### ISSUE TITLE
write a how to debug mailing issues

### ISSUE THREAD

**Issue by noave** [Uberspace/manual#488]
- check for .qmail files
- check spamfolder status
- check sieve status
- check MX records
- check spf, dkim 
- show what we need for support questions: specific mailaddresses, timestamps within last two days for logs

**Comment by quite** [comment]
Yeah a way to get logs about what sieve decided to do with a mail would be good... like legacy ~/var/log/mailfilter.log
